src/libraries/System.Diagnostics.DiagnosticSource/src/Resources/Strings.resx

```diff
@@ -165,4 +165,7 @@
   <data name="UnableToInitialize" xml:space="preserve">
     <value>Unable to initialize all required reflection objects</value>
   </data>
+  <data name="UnsupportedType" xml:space="preserve">
+    <value>{0} is unsupported type for this operation. The only supported types are byte, short, int, long, float, double and decimal.</value>
```

Comment:
Nit:
`double, and decimal`
I'm an Oxford comma guy :)

Reply:
Ah, then you may like this discussion: https://github.com/dotnet/runtime/pull/52554#discussion_r630544416

Reply:
Yeah, two spaces after a period is another good one.  But the Oxford comma can actually help avoid ambiguities in text, whereas my preference for two spaces is purely about visual comfort.

Reply:
Agreed